### PR TITLE
Add exploration asset view

### PR DESCRIPTION
**Pull request by petterwildhagen**

### CODE REVIEW

File: frontend/src/models/assets/exploration/Exploration.ts [hunk 1]
```diff
@@ -7,20 +7,42 @@ export class Exploration implements Components.Schemas.ExplorationDto {
...
+            this.name = ""
+            this.costProfile = new ExplorationCostProfile()
+        }
+    }
+
+    static Copy(data: Exploration) {
+        const transportCopy = new Exploration(data)
+        return {
+            ...transportCopy,
+            costProfile: data.costProfile,
```
[the remark this answers is not in the document]
Comment (taustad) [reply frontend/src/models/assets/exploration/Exploration.ts:36]: transportCopy should probably be explorationCopy

**Review by taustad — COMMENTED** [review]
Looks good

**Review by taustad — APPROVED** [review]
Approved

File: frontend/src/models/assets/exploration/Exploration.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (petterwildhagen) [reply frontend/src/models/assets/exploration/Exploration.ts:36]: Fixed

**Review by taustad — APPROVED** [review]

**Pull request merged**